packages/components/src/components/input/input.component.ts

```diff
@@ -539,6 +570,40 @@ export default class SynInput extends SynergyElement implements SynergyFormContr
             <span part="suffix" class="input__suffix">
               <slot name="suffix"></slot>
             </span>
+
+            ${this.type === 'number' && !this.noSpinButtons
```

Comment:
I wouldn't do anything here. It's the same behaviour for default HTML inputs.

https://github.com/synergy-design-system/synergy-design-system/assets/26542182/149226d4-9035-4e4b-9d3c-5d3db1a776a8

And I wouldn't try to fix something, which could lead to other problems later on. This is such a super edge case (having 0.5 and using steppers instead of keayboard and not typing anything etc.)